src/lib/components/input-with-icon/InputWithIcon.svelte

```diff
@@ -1,57 +1,60 @@
 <script lang="ts">
-	import type { HTMLInputTypeAttribute } from '../../../types/html-input-type-attribute.type'
+  import type { HTMLInputTypeAttribute } from '../../../types/html-input-type-attribute.type'
```

Comment:
```suggestion
  import type { HTMLInputTypeAttribute } from '$lib/types/html-input-type-attribute.type'
```
Claro, esto implica mover el html-input-type-attribute a types en lib

Reply:
arreglao